nuxeo-dev-apb/apb.yml

```diff
@@ -109,81 +109,97 @@ plans:
       - name: backend_repository_type
         default: 'DBS'
         type: enum
-        enum: ['DBS', 'VCS_Postgres']
+        # TODO: Make Postgres work before showing it as available.
+        # enum: ['DBS', 'VCS_Postgres']
+        enum: ['DBS']
         title: Repository type
         display_group: Backend storages
 
       - name: backend_stream_type
         default: 'Chronicle'
         type: enum
-        enum: ['Chronicle', 'Kafka']
+        # TODO: Make Kafka work before showing it as available.
+        # enum: ['Chronicle', 'Kafka']
+        enum: ['Chronicle']
         title: Stream type
         display_group: Backend storages
 
       - name: backend_pubsub_type
         default: inmemory
         type: enum
-        enum: ['inmemory', 'stream', 'redis']
+        # TODO: Make Stream and Redis work before showing them as available.
+        # enum: ['inmemory', 'stream', 'redis']
+        enum: ['inmemory']
         title: Stream type
         display_group: Backend storages
 
       - name: backend_binaries_type
         default: fs
         type: enum
-        enum: ['fs', 'minio', 's3']
+        # TODO: Make Minio and S3 work before showing them as available.
+        # enum: ['fs', 'minio', 's3']
+        enum: ['fs']
         title: Binaries storage type
         display_group: Backend storages
 
       # TODO: K/V store type
 
       - name: backend_pv_storage_class
-        type: string
         title: Storage class for backend persistence volumes
-        default: gp2
+        # TODO: Make other backend storage classes work before showing this field as a free text field.
```

Comment:
Storage class must be kept free-from as they are declared at the Openshift level and as they are not managed by this APB

Reply:
ok, fixed